extensions/cache/src/main/resources/benchmark-stream-operations.xml

```diff
@@ -0,0 +1,132 @@
+<!-- RadarGun 3.0 benchmark -->
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <!-- Specifies where should the master open socket -->
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}" />
+
+   <!-- List of cluster configurations where the benchmark should run -->
+   <clusters>
+      <cluster size="2">
+         <group name="server" size="1" />
+         <group name="client" size="1" />
+      </cluster>
+
+   </clusters>
+
+   <!-- List of configurations of the services -->
+   <configurations>
+
+      <config name="ISPN9 - hotrod">
+         <setup group="server" plugin="infinispan90">
+            <server xmlns="urn:radargun:plugins:infinispan90:3.0" file="streaming.xml"
+               jmx-domain="jboss.datagrid-infinispan" start-timeout="120000" cache-manager-name="clustered">
+               <args>
+                  -Djava.net.preferIPv4Stack=true
+                  -Djboss.node.name=node${slave.index}
+                  -Djboss.socket.binding.port-offset=${slave.index}00
+               </args>
+               <home>${RADARGUN_WORK}/slave${slave.index}</home>
+               <server-zip>${ISPN_ZIP_PATH}/infinispan-server-9.0.0-SNAPSHOT.zip</server-zip>
+               <env>JAVA_OPTS=-server -Xms2g -Xmx4g -XX:NewRatio=3
```

Comment:
I am pretty sure I am demonstrating a use case. Anyway, I left this configuration there specifically so there is some example of how to do this. I was unable to find one when I was starting this and would be grateful for one.

I will be more than happy to remove it if you can point me to a example I was unable to find. Otherwise I will remove it, but not be happy.